deploy/modules/aws/tidb-operator/manifests/local-volume-provisioner.yaml

```diff
@@ -12,6 +12,8 @@ metadata:
   name: local-provisioner-config
   namespace: kube-system
 data:
+  nodeLabelsForPV: |
```

Comment:
This copies the label from the node to the PV, but what is the purpose of that?

Reply:
Users can simply know the PV belongs to which node or a node contains which PV by `kubectl get pv --show-labels | grep <node-name>`.